lib/components/tabs.js

```diff
@@ -26,6 +26,7 @@ export default class Tabs extends Component {
             <div className={css('title')}>{tabs[0].title}</div> :
             [
               <ul
+                key="tabs-container"
```

Comment:
It's not really a component though, just a list of `Tab` components. I think `tabs-list` might be better?